packages/eas-cli/src/project/isEasEnabledForProject.ts

```diff
@@ -0,0 +1,21 @@
+import { apiClient } from '../api';
+
+/**
+ * Checks if the project is allowed for using EAS services.
+ * THIS IS A TEMPORARY STEP NEEDED FOR OPEN PREVIEW
+ * @returns boolean
+ */
+export async function isEasEnabledForProjectAsync(projectId: string): Promise<boolean> {
+  try {
+    const {
+      data: { enabled },
+    } = await apiClient.get(`projects/${projectId}/eas-enabled`).json();
```

Comment:
I commented in the server side PR – would be good to get this information as a part of the GraphQL query we make in `getProjectIdAsync`. I know this is temporary, but since we will do this everytime you run a command, fetching it in the same request would make the CLI more snappy.

Reply:
Done